src/storage/compactor/src/lib.rs

```diff
@@ -89,6 +89,15 @@ pub struct CompactorOpts {
     #[clap(long, env = "RW_OBJECT_STORE_READ_TIMEOUT_MS", value_enum)]
     #[override_opts(path = storage.object_store_read_timeout_ms)]
     pub object_store_read_timeout_ms: Option<u64>,
+
+    pub is_shared_compactor: bool,
+    endpoint: String,
+    parallel_compact_size_mb: u32,
+    sstable_size_mb: u32,
+    block_size_kb: u32,
+    bloom_false_positive: f64,
```

Comment:
Do we need to provide this global settings to the compactor? I thought it is controlled by compaction group configs and are set per task. cc @Li0k

Reply:
After some offline discussion, we decide to reuse system params, and to not put these global settings in `CompactorOpts`, every time starting a new compactor pod, it will get system params via `SystemParamsServiceClient`.